dev/sg/internal/rfc/rfc.go

```diff
@@ -55,7 +61,7 @@ func getClient(ctx context.Context, config *oauth2.Config, out *std.Output) (*ht
 	if err := sec.Get("rfc", tok); err != nil {
 		// ...if it doesn't exist, open browser and ask user to give us
 		// permissions
-		tok, err = getTokenFromWeb(ctx, config, out)
+		tok, err = getTokenFromWeb(handleAuthResponse, ctx, NewTokenHandler(config), out)
```

Comment:
nit:i believe by convention ctx should be passed in as the first argument 
```suggestion
		tok, err = getTokenFromWeb( ctx,handleAuthResponse, NewTokenHandler(config), out)
```

Reply:
@mucles that's correct